iOS/ViewController.swift

```diff
@@ -69,6 +70,12 @@ class ViewController: UIViewController, SpeakerDelegate, UIPickerViewDelegate, U
         present(alertController, animated: true, completion: nil)
     }
     
+    @IBAction func rateChanged(_ sender: UISlider) {
+        let value = roundf(100 * sender.value) / 100
```

Comment:
Thank you for reviewing @eeneguee !. I tried to divided it into 10 levels but it didn't look natural...  The rate was changed too much.